### PR TITLE
Removed v1 contracts

### PR DESCRIPTION
**Pull request by TrevorJTClarke**
Removing the contracts in favor of less confusion, migrated the v1 contracts to the new [Legacy Contracts](https://github.com/near-daos/sputnik-dao-contract-legacy) repo

### CODE REVIEW

**Comment by TrevorJTClarke** [comment]
NOTE: @ctindogaru this needs a solid build, was running into deps version issues.

**Comment by TrevorJTClarke** [comment]
@constantindogaru Ready for review

**Comment by ctindogaru** [comment]
It's so good to finally see this.

However, I'm not sure why `sputnikdao-factory2/Cargo.lock` and `sputnikdao2/Cargo.lock` got deleted. Was this intended?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Sputnik DAO v2
 
-> Building on the functionality of [v1](sputnikdao/README.md), Sputnik DAO v2 offers even more features and enhanced configuration ability.
+> Building on the functionality of [Sputnik V1](https://github.com/near-daos/sputnik-dao-contract-legacy), Sputnik DAO v2 offers even more features and enhanced configuration ability. Sputnik V1 is archived, all Sputnik V2 features can be upgraded upon new releases inside the deployed factory.
```
[the remark this answers is not in the document]
Comment (TrevorJTClarke) [reply README.md:3]: Good point on consistency - will update

**Review by ctindogaru — APPROVED** [review]
nicely done!

**Pull request merged**